### ISSUE TITLE
Feature Request: Allow styling attributes to be included during dynamic component creation.

### ISSUE THREAD

**Issue by patrickseda** [NativeScript/NativeScript#1758]
When dynamically creating custom components via `builder.load(LoadOptions)`, it is not possible to pass in styling attributes to be used during creation. Sometimes we want styling values to be created dynamically and not be pulled from a CSS file.

Internal method `loadCustomComponent()` has a parameter available (named `attributes`), but it is not used when called via `builder.load()`, rather it is hardcoded to `undefined`. Perhaps this could be exposed so we could take advantage of it?

Currently we must set any dynamic styling values post-creation like:

```
var customComponent = builder.load({
    path: '~/',
    name: 'customComponent'
});
customComponent.marginTop = 10;
customComponent.marginLeft = 30;
```

But it would be nice to have a single call for creation and styling like:

```
var customComponent = builder.load({
    path: '~/',
    name: 'customComponent',
    attributes: {
        marginTop: 10,
        marginLeft: 30
    }
});
```

**Comment by patrickseda** [comment]
This is being addressed with PR: https://github.com/NativeScript/NativeScript/pull/2049

**Comment by atanasovg** [comment]
This is closed with #2049
